CMakeLists.txt

```diff
@@ -24,41 +24,53 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 
+# New escape-sequence processing.
+cmake_minimum_required(VERSION 3.5)
+cmake_policy(VERSION 3.5.1)
 
 # Don't check any language compiler. This project is for EditorConfig Core
 # testing only.
 project(editorconfig-core-test NONE)
 
 # Where this file lives
 set(tests_cmakelists_dir "${CMAKE_CURRENT_LIST_DIR}")
-#message(STATUS "Tests are in ${tests_cmakelists_dir}")
+message(STATUS "Tests are in ${tests_cmakelists_dir}")
 
 # Only when we are using editorconfig-core-test independently should we check
 # cmake version, set EDITORCONFIG_CMD as cache string, and enable_testing()
 # here.
 if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
-    cmake_minimum_required(VERSION 3.5)
     set(EDITORCONFIG_CMD "editorconfig" CACHE STRING "editorconfig command.")
+    set(EDITORCONFIG_CMD_IS_TARGET OFF CACHE BOOL
+        "EDITORCONFIG_CMD names a CMake target rather than an executable.")
     enable_testing()
+    message(STATUS "tests: Standalone testing enabled")
+endif()
+
+if(${EDITORCONFIG_CMD_IS_TARGET})
+    string(REGEX REPLACE "^=" "" EDITORCONFIG_CMD ${EDITORCONFIG_CMD})
```

Comment:
Is there a reason we still require a target name starts with `=`? Would `EDITORCONFIG_CMD_IS_TARGET` be sufficient?